be/src/common/config.cpp

```diff
@@ -281,6 +281,8 @@ DEFINE_mInt32(doris_scan_range_max_mb, "1024");
 DEFINE_mInt32(doris_scanner_row_num, "16384");
 // single read execute fragment row bytes
 DEFINE_mInt32(doris_scanner_row_bytes, "10485760");
+// single read execute fragment max run time millseconds
+DEFINE_mInt32(doris_scanner_max_run_time_ms, "1000");
```

Comment:
change it to session variable?

Reply:
I think it is a system configuration, similar with `doris_scanner_row_bytes` or `doris_scanner_row_num`, should not changed by session level.